### ISSUE TITLE
Warning thrown with ansible.builtin.iptables examples

### ISSUE THREAD

**Issue by imjoseangel** [ansible/ansible#74118]
### Summary

When throwing `ansible.builtin.iptables` example like this:

```yaml
    - name: Forward port 80 to 8600
      ansible.builtin.iptables:
        table: nat
        chain: PREROUTING
        in_interface: eth0
        protocol: tcp
        match: tcp
        destination_port: 80
        jump: REDIRECT
        to_ports: 8600
        comment: Redirect web traffic to port 8600
      become: yes
```

Ansible complaints about value type:

```
TASK [Forward port 80 to 8600] ********************************************************************************************************************************************************************************************************
[WARNING]: The value "80" (type int) was converted to "'80'" (type string). If this does not look like what you expect, quote the entire value to ensure it does not change.
[WARNING]: The value "8600" (type int) was converted to "'8600'" (type string). If this does not look like what you expect, quote the entire value to ensure it does not change.
ok: [localhost]
```

### Issue Type

Bug Report

### Component Name

iptables

### Ansible Version

```console (paste below)
$ ansible --version
ansible 2.10.7
  config file = None
  configured module search path = ['/home/imjoseangel/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/imjoseangel/.pyenv/versions/3.8.9/lib/python3.8/site-packages/ansible
  executable location = /home/imjoseangel/.pyenv/versions/3.8.9/bin/ansible
  python version = 3.8.9 (default, Apr  4 2021, 11:33:43) [GCC 7.5.0]
```

### Configuration

```console (paste below)
$ ansible-config dump --only-changed
INTERPRETER_PYTHON(env: ANSIBLE_PYTHON_INTERPRETER) = /home/imjoseangel/.pyenv/shims/python3
```

### OS / Environment

```bash
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.5 LTS
Release:	18.04
Codename:	bionic
```

### Steps to Reproduce

```yaml
    - name: Forward port 80 to 8600
      ansible.builtin.iptables:
        table: nat
        chain: PREROUTING
        in_interface: eth0
        protocol: tcp
        match: tcp
        destination_port: 80
        jump: REDIRECT
        to_ports: 8600
        comment: Redirect web traffic to port 8600
      become: yes
```

### Expected Results

No warning thrown in the examples or value converted in the modules

### Actual Results

```console (paste below)
TASK [Forward port 80 to 8600] ********************************************************************************************************************************************************************************************************
[WARNING]: The value "80" (type int) was converted to "'80'" (type string). If this does not look like what you expect, quote the entire value to ensure it does not change.
[WARNING]: The value "8600" (type int) was converted to "'8600'" (type string). If this does not look like what you expect, quote the entire value to ensure it does not change.
```

**Comment by imjoseangel** [comment]
Following #74114, I will check the module and review these behaviors

**Comment by s-hertel** [comment]
This is fixed in 2.11.
